src/app/app.component.spec.ts

```diff
@@ -20,16 +23,16 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'ui-docs-site'`, () => {
+  it(`should have as title 'forge-site'`, () => {
```

Comment:
Are these tests still valid?

Reply:
Doubtful. I didn't address the tests in this ticket.

Reply:
If they're not used I say we get rid of them

Reply:
follow-up... this one is used, it's the next one, with the <h1> that's not.